Miscellaneous improvements to the Queues docs (#6441)
cloudflare/cloudflare-docs: content/queues/configuration.md

Each Worker has a `wrangler.toml` configuration file that specifies environment variables, triggers, and resources, such as a Queue. Use the options below to configure your Queue.

{{<Aside type="warning" header="Warning">}}

Queues support is not yet available in a full Wrangler release. It will be soon, but in the meantime please instead install the `wrangler@queues` npm package by running `npm install -D wrangler@queues` within your project.

{{</Aside>}}

{{<Aside type="note">}}

> Queues are currently only configurable via Wrangler. Support for configuring Queues in the Dashboard will be coming soon.

Reviewer: ```suggestion
Queues are currently only configurable via [Wrangler](/workers/wrangler/get-started/). Support for configuring Queues in the dashboard will be coming soon.
```

## Consumer

> These options should be used when a Worker wants to receive messages from a Queue. Maximum values for each of these options can be found on the [Limits](/queues/limits) page.

Reviewer: ```suggestion
These options should be used when a Worker wants to receive messages from a Queue. Refer to [Limits](/queues/limits) to review the maximum values for each of these options.
```
